assets/javascript/app.js

```diff
@@ -1,4 +1,43 @@
 //= require_directory ./vendor
 //
 
-// custom JS can go in this file, or another, then you add another require line
\ No newline at end of file
+// custom JS can go in this file, or another, then you add another require line
+
+function showCareerPrev(currIndex) {
+  'use strict';
```

Comment:
You can put 'use strict'; at the top of the file instead of including it in every function. It'll work the same.